ci/test_wheel_cudf.sh

```diff
@@ -3,19 +3,23 @@
 
 set -eou pipefail
 
+# Download the pylibcudf built in the previous step
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+RAPIDS_PY_WHEEL_NAME="pylibcudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./local-pylibcudf-dep
 RAPIDS_PY_WHEEL_NAME="cudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
 
-# echo to expand wildcard before adding `[extra]` requires for pip
-python -m pip install $(echo ./dist/cudf*.whl)[test]
+# Install both pylibcudf and cudf
+python -m pip install \
+    "$(echo ./local-pylibcudf-dep/pylibcudf*.whl" \
```

Comment:
Missing closing parenthesis and test deps.
```suggestion
    "$(echo ./local-pylibcudf-dep/pylibcudf*.whl)[test]" \
```